docs/sql-ref-syntax-qry-select-having.md

```diff
@@ -18,5 +18,101 @@ license: |
   See the License for the specific language governing permissions and
   limitations under the License.
 ---
+The <code>HAVING</code> clause is used to filter the results produced by
+<code>GROUP BY</code> based on the specified condition. It is often used
+in the conjunction with a <code>GROUP BY</code> clause.
 
-**This page is under construction**
+### Syntax
+{% highlight sql %}
+HAVING boolean_expression
+{% endhighlight %}
+
+### Parameters
+<dl>
+  <dt><code><em>boolean_expression</em></code></dt>
+  <dd>
+    Specifies any expression that evaluates to a result type <code>boolean</code>. Two or
+    more expressions may be combined together using comparision operators 
+    ('>', '>=', ‘>’, ‘>=’, ‘=’, ‘<’ and ‘<=’, '<=>') and logical operators (AND, NOT, OR).<br><br>
```

Comment:
This doesn't look right:
![image](https://user-images.githubusercontent.com/13592258/72696171-2117ee80-3af0-11ea-884e-d983ded445fc.png)